### ISSUE TITLE
how to define masters and workers in the hosts file

### ISSUE THREAD

**Issue by simonjcarr** [vandot/ansible-k3sup#4]
What is ansible-k3sup looking for in the hosts file to identify which servers are masters and which are workers?

**Comment by vandot** [comment]
In the README [example](https://github.com/vandot/ansible-k3sup#Example) fresh new servers are created on DO and later `k3sup` is used to spin up master and workers, so no hosts file.
You have existing servers where you want to install `k3s` cluster?

**Comment by dragid10** [comment]
> You have existing servers where you want to install `k3s` cluster?

Yeah if I'm setting up a raspberry pi cluster, I'd want to define some additional properties via K3sup
